### PR TITLE
make network vlan searchable

### PR DESCRIPTION
**Pull request by ahaupt**
It would be much more convenient for us to directly search a network by its VLAN tag. This simple patch extends the list of searchable items by the vlan key.

### CODE REVIEW

File: src/ralph/networks/admin.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def get_queryset(self, request):
 class NetworkAdmin(RalphMPTTAdmin):
     ordering = ['min_ip', '-max_ip']
     change_form_template = 'admin/data_center/network/change_form.html'
-    search_fields = ['name', 'address', 'remarks']
+    search_fields = ['name', 'address', 'remarks', 'vlan']
```
Comment (romcheg) [inline src/ralph/networks/admin.py:114]: Could you please also make it possible to filter by VLAN range by extending ```list_filter``` property with ```'vlan'```? You may put it into any position that you think will be the best.

**Review by romcheg — CHANGES_REQUESTED** [review]
Thanks for your contribution! I have mentioned one small change inline, otherwise LGTM!

File: src/ralph/networks/admin.py (hunk 1 above)
Comment (ahaupt) [reply src/ralph/networks/admin.py:114]: OK, should be in 37415d16bc04fbe41bcf4fe754131b601eb234b0

Comment (romcheg) [reply src/ralph/networks/admin.py:114]: Thanks!

**Review by romcheg — APPROVED** [review]
I verified the changes on my local environment and can confirm everything works as expected. Thanks!

**Review by szymi- — APPROVED** [review]

**Pull request merged**